### PR TITLE
add back ssl

### PR DESCRIPTION
**Pull request by f-w**

### CODE REVIEW

**Review by mark-walle — COMMENTED** [review]
@f-w I'm not a VSCode user so I'm just unfamiliar with the intent of the `.vscode/launch.js` configuration file. Is it conventional to include that into repositories? Will this support other developers on this project who use VSCode?

**Review by mark-walle — COMMENTED** [review]
@f-w Re: `.pipeline/lib/config.js` nothing in the functionality is actually changed here, is it?

Otherwise, thanks for the cleanup on it.

**Review by mark-walle — APPROVED** [review]
The rest of these changes re-implement the SSL server for HTTPS and passthrough. Looks good to me. Thanks @f-w

**Comment by f-w** [comment]
> @f-w I'm not a VSCode user so I'm just unfamiliar with the intent of the `.vscode/launch.js` configuration file. Is it conventional to include that into repositories? Will this support other developers on this project who use VSCode?

It's intended for sharing debug config. It doesn't contain any sensitive data so I don't see it to be a problem. For non vs code user, you can just ignore it.

**Comment by f-w** [comment]
> @f-w Re: `.pipeline/lib/config.js` nothing in the functionality is actually changed here, is it?
> 
> Otherwise, thanks for the cleanup on it.

The non-formatting change I made in that file is to replace host: '*caps.pathfinder.gov.bc.ca' with host: '*caps.analytics.gov.bc.ca',

File: .pipeline/lib/config.js [hunk 1]
```diff
@@ -1,14 +1,56 @@
...
+  dev: {
+    namespace: '8gsiqa-dev',
+    name: `${name}`,
+    phase: 'dev',
+    changeId: changeId,
+    suffix: `-dev-${changeId}`,
+    instance: `${name}-dev-${changeId}`,
+    version: `${version}-${changeId}`,
+    tag: `dev-${version}-${changeId}`,
+    host: 'dev-caps.analytics.gov.bc.ca',
```
Comment (mark-walle) [inline .pipeline/lib/config.js:27]: I see that `*.caps.pathfinder` hosts are now changed to `*.analytics.gov.bc.ca` which makes sense per the intent to get SSL signed for all caps environments. Thanks for clarifying that @f-w

**Pull request closed without merging**